test/data/mock-state.json

```diff
@@ -383,20 +383,30 @@
       "unconnectedAccount": true
     },
     "featureFlags": {},
-    "networkConfigurations": {
-      "testNetworkConfigurationId": {
-        "rpcUrl": "https://testrpc.com",
+    "networkConfigurationsByChainId": {
+      "0x1": {
         "chainId": "0x1",
-        "nickname": "Custom Mainnet RPC",
-        "type": "rpc",
-        "id": "testNetworkConfigurationId"
+        "name": "Custom Mainnet RPC",
+        "nativeCurrency": "ETH",
+        "defaultRpcEndpointIndex": 0,
+        "rpcEndpoints": [
+          {
+            "url": "https://testrpc.com",
+            "networkClientId": "testNetworkConfigurationId"
+          }
+        ]
       },
-      "goerli": {
-        "type": "rpc",
+      "0x5": {
         "chainId": "0x5",
-        "ticker": "ETH",
-        "nickname": "Chain 5",
-        "id": "goerli"
+        "name": "Goerli",
+        "nativeCurrency": "ETH",
+        "defaultRpcEndpointIndex": 0,
+        "rpcEndpoints": [
+          {
+            "networkClientId": "goerli",
+            "type": "custom"
```

Comment:
is this type correct? or should it have a url property?

Reply:
url is a required property and would always be defined in the real state.  I'll try to add one.

Reply:
https://github.com/MetaMask/metamask-extension/pull/26433/commits/838529c5d346a0cd1e3b0e4bc2fdb5fa76a90b40